server/handlers.go

```diff
@@ -1034,6 +1034,11 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 
 	if action == "inline" {
 		disposition = "inline"
+		/*
+		metadata.ContentType is unable to determine the type of the content, 
+		So add text/plain in this case to fix XSS related issues/
+		*/
+		contentType := "text/plain"
```

Comment:
let's override `contentType` only if it's empty, and in case let's fix what's missing in https://github.com/dutchcoders/transfer.sh/blob/64c7759126ffea425c55d76dd49cbbc1a6719992/server/handlers.go#L97-L116

thanks

Reply:
or maybe:
https://github.com/dutchcoders/transfer.sh/blob/main/server/handlers.go#L1051
```
if (disposition == "inline" && canContainsXSS(contentType)) || (contentType == "") {
```


@stefanbenten what do you thini it's the best approach?